examples/treeple/treeple_tutorial_CMI.py

```diff
@@ -1,18 +1,24 @@
 """
-====================================
-Treeple tutorial for calculating CMI
-====================================
+=====================
+2-1b: Calculating CMI
+=====================
 """
 
 import matplotlib.pyplot as plt
 import numpy as np
+import pandas as pd
+import seaborn as sns
 from scipy.stats import entropy
 
 from sktree.datasets import make_trunk_classification
 from sktree.ensemble import HonestForestClassifier
 from sktree.stats import build_hyppo_oob_forest
 from sktree.tree import MultiViewDecisionTreeClassifier
 
+sns.set(color_codes=True, style="white", context="talk", font_scale=1.5)
```

Comment:
Maybe add a comment above stating we are using some formatting to make the figure look as close to "TODO" insert citation to the paper.

Reply:
I don't feel that's necessary. It's just a figure style? Definitely agree on citing the paper somewhere when it's out.